### PR TITLE
feat(interview-prep): Porting Rosetta Problem: Sort stability

### PR DESCRIPTION
**Pull request by NitronR**
- [ ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by nikrb** [comment]
@RandellDawson successfully tested locally

**Comment by nikrb** [comment]
@NitronR a large part of the description seems to be copied from [here](https://rosettacode.org/wiki/Sort_stability). We do not condone plagiarism so please update or attribute to the source.

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-stability.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## Instructions
+<section id='instructions'>
+
+</section>
+## Tests
+<section id='tests'>
+
+``` yml
+tests:
```
Comment (scissorsneedfoodtoo) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-stability.md]: Could you indent the tests like so?:

```js
tests:
  - text: <code>stableSort</code> should be a function.
    testString: assert(typeof stableSort=='function','<code>stableSort</code> should be a function.');
```

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-stability.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+</section>
+
+## Challenge Seed
+<section id='challengeSeed'>
+<div id='js-seed'>
+
+```js
+function stableSort (arr) {,
```
Comment (scissorsneedfoodtoo) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sort-stability.md]: Seems to be a couple of unnecessary commas here:

```js
function stableSort (arr) {
  // Good luck!
}
```

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]
We should be good to go once those two things are fixed!

**Comment by NitronR** [comment]
@scissorsneedfoodtoo Removed the commas and fixed indentation in all 6 PRs where needed.

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Comment by scissorsneedfoodtoo** [comment]
Everything LGTM! Thanks for all of your hard work, @NitronR.

**Pull request merged**